k8s/security/admission-controllers/opa-gatekeeper.yaml

```
@@ -0,0 +1,315 @@
+      validation:
+        type: object
+        properties:
+          limits:
+            type: object
+            properties:
+              cpu:
+                type: string
+              memory:
+                type: string
+  targets:
+    - target: admission.k8s.gatekeeper.sh
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)